assets/js/components/dashboard/EventForm.tsx

```diff
@@ -0,0 +1,162 @@
+
+import { EventFormType } from "../../types";
+
+import { UnorderedList, ListItem } from "@chakra-ui/react";
+import GenericButton from "../helpers/GenericButton";
+
+import "../../../css/dashboard/eventform.css";
+
+export const initialEventFormInput: EventFormType = {
+  title: "",
+  description: "",
+  start_date: "",
+  presenters: [],
+};
+
+type FieldProps = {
+  inputSetter: (fieldInput: string) => void;
+};
+
+const changeElement = (
+  event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>,
+  inputSetter: (fieldInput: string) => void
+) => {
+  return inputSetter(event.target.value);
+};
+
+const TitleField = ({ inputSetter }: FieldProps) => {
+  return (
+    <div className="EventFormFieldDiv">
+      <label className="EventFormFieldLabel">Event name</label>
+      <input
+        className="EventFormFieldInput"
+        type="text"
+        placeholder="Type title here"
+        onChange={(e) => changeElement(e, inputSetter)}
+      />
+    </div>
+  );
+};
+
+const DescriptionField = ({ inputSetter }: FieldProps) => {
+  const charLimit = 255;
+  const [counter, setCounter] = useState<number>(0);
+
+  const handleChange = (event: React.ChangeEvent<HTMLTextAreaElement>): void => {
+    const numberOfChars: number = event.target.value.length;
+    setCounter(numberOfChars);
+
+    changeElement(event, inputSetter);
+  };
+
+  return (
+    <div className="EventFormFieldDiv">
+      <div className="FlexContainer">
+        <label className="EventFormFieldLabel">Event description</label>
+        <span>
+          {counter}/{charLimit}
+        </span>
+      </div>
+      <textarea
+        className="EventFormFieldInput"
+        id="EventFormDescriptionField"
+        placeholder="Type description here"
+        onChange={handleChange}
+      ></textarea>
+    </div>
+  );
+};
+
+const DateField = ({ inputSetter }: FieldProps) => {
+  return (
+    <div className="EventFormFieldDiv">
+      <label className="EventFormFieldLabel">Date & Time</label>
+      <input
+        className="EventFormFieldInput"
+        type="datetime-local"
+        onChange={(e) => changeElement(e, inputSetter)}
+      />
+    </div>
+  );
+};
+
+type ListFieldProps = { inputList: string[]; inputSetter: (inputList: string[]) => void };
+
+const PresenterField = ({ inputList, inputSetter }: ListFieldProps) => {
+  const [inputPresenter, setInputPresenter] = useState<string>("");
+
+  const handleAddButton = () => {
+    inputSetter([...inputList, inputPresenter]);
+    setInputPresenter("");
+  };
+
+  return (
+    <div className="EventFormFieldDiv">
+      <label className="EventFormFieldLabel">Choose presenters (optional)</label>
+      <div className="EventFormFieldInput EventFormFieldWithButtonDiv">
+        <input
+          type="search"
+          placeholder="Type name of presenters to add"
+          onChange={(e) => changeElement(e, setInputPresenter)}
+        />
+        <GenericButton icon={<Search />} onClick={handleAddButton} />
```

Comment:
As I mentioned while chatting, I think it's better to use some kind of "plus" icon instead of `Search`.

Reply:
Right, I'll change that

Reply:
Changed to a plus